docs/man/docker.1.md

```diff
@@ -34,6 +34,9 @@ unix://[/path/to/socket] to use.
    The socket(s) to bind to in daemon mode specified using one or more
    tcp://host:port, unix:///path/to/socket, fd://* or fd://socketfd.
 
+**--add-registry**=[]
+  Each given registry will be prepended to a list of registries queried during image pulls or searches. The last registry given will be queried first. They will be treated as insecure. Registry mirrors won't apply to them.
```

Comment:
LIFO policy in this case seems more natural to me. If you consider the list of queried registries as a list with one initial value `docker.io`, then each addition inserts registry on the first position. FIFO policy, on the other hand, inserts to a last but one position - before `docker.io`, which is kind of awkward to imagine and explain. But I agree it's confusing that these similar flags behave differently. I guess nothing can be done about `--registry-mirror` now. So I'll make `--add-registry` behave the same.